src/app/setup-wizard/data-collection/on-site-assessment/assessment-details-form/assessment-details-form.component.ts

```diff
@@ -67,26 +67,26 @@ export class AssessmentDetailsFormComponent {
   }
 
   async assessmentTypeChange() {
-    let utilityTypes = AssessmentOptions.find(
-      _assessmentOption => _assessmentOption.assessmentType == this.assessment.assessmentType)?.utilityTypes || [];
-    this.assessment.utilityTypes = utilityTypes; // track all utility types
-    // utility type is not the default if the assessment type changes
-    if (this.assessment.utilityType !== utilityTypes[0]) {
-      this.assessment.utilityType = utilityTypes?.[0]; // update to the first/default utility type
-      await this.utilityTypeChange();
-    } else {
-      await this.saveChanges();
-    }
+    // let utilityTypes = AssessmentOptions.find(
```

Comment:
If we don't need this commented code we can delete it, unless we know more logic is coming later then we should tag the issue that it will be brought back for

Reply:
yes, the code was commented because this assessment details form was also affected by the changes in pre-assessment. I rolled back some changes, so this assessment details form will still work (will be updated in another issue). Thanks!